src/sentry/api/endpoints/event_owners.py

```diff
@@ -44,5 +44,8 @@ def get(self, request, project, event_id):
                 request.user,
                 ActorSerializer(),
             ),
-            'rule': matcher,
+            # TODO(mattrobenolt): We need to change the API here to return
+            # all rules, just keeping this way currently for API compat
+            'rule': rules[0].matcher if rules else None,
+            'rules': rules or [],
```

Comment:
@MaxBittker If you wanna take this bit over and leverage this more accurate data in the tooltip in the UI, that'd be cool.

Reply:
I think me/lyn will probably take this on so Max can do integrations stuff?

Reply:
wfm, I just know he built the original version.